examples/CMIP6_ScenarioMIP_NCAR_CESM2.json

```diff
@@ -0,0 +1,58 @@
+{
+  "type": "Feature",
+  "stac_version": "1.1.0",
+  "id": "CMIP6_ScenarioMIP_NCAR_CESM2",
+  "properties": {
+    "start_datetime": "2015-01-01T00:00:00Z",
+    "end_datetime": "2024-12-31T23:59:59Z",
+    "cube:dimensions": {
+      "time": {
+        "type": "temporal",
+        "extent": ["2015-01-01T00:00:00Z", "2024-12-31T23:59:59Z"]
+      },
+      "lat": {
+        "type": "spatial",
+        "axis": "y",
+        "extent": [-90, 90]
+      },
+      "lon": {
+        "type": "spatial",
+        "axis": "x",
+        "extent": [-180, 180]
+      }
+    }
+  },
+  "links": [
+    {
+      "rel": "store",
+      "href": "https://storage.example.com/kerchunk/CMIP6_CESM2_rlus_kr1.0.json",
+      "type": "application/json+zstd",
+      "title": "Kerchunk Reference File"
+    }
+  ],
+  "assets": {
+    "rlus": {
+      "href": "https://storage.example.com/zarr-refs/CMIP6_CESM2_rlus.json",
+      "type": "application/vnd+zarr; version=2",
+      "title": "Surface Upwelling Longwave Radiation",
+      "roles": ["data"],
+      "cube:variables": {
+        "rlus": {
+          "type": "data",
+          "dimensions": ["time", "lat", "lon"],
+          "unit": "W m-2",
+          "attrs": {
+            "long_name": "Surface Upwelling Longwave Radiation",
+            "standard_name": "surface_upwelling_longwave_flux_in_air"
+          }
+        }
+      }
+    },
+    "source_data": {
+      "href": "https://storage.example.com/data/CMIP6/rlus_day_CESM2-WACCM_ssp585_gn_20150101-20241231.nc",
+      "type": "application/netcdf",
+      "title": "Original NetCDF file",
+      "roles": ["source"]
```

Comment:
The source data is unlikely to be a single file. It is more useful to point to a bucket that contains a whole bunch of netcdf files. For icechunk you need the path to that bucket in order to authenticate properly and be able to access the chunks of data.